code/__DEFINES/surgery.dm

```diff
@@ -24,6 +24,8 @@
 #define ORGAN_UNREMOVABLE (1<<8)
 /// Can't be seen by scanners, doesn't anger body purists
 #define ORGAN_HIDDEN (1<<9)
+/// Has the organ already been used by somebody once before
+#define ORGAN_VIRGIN (1<<10)
```

Comment:
this is not a really good comment, an organ being virgin means precisely the opposite (it has yet to be used by anyone)

Reply:
https://github.com/tgstation/tgstation/pull/78519/commits/2a73724a38542174c653ea99d48c641d043a1c50